src/adapters/mc/README.md

```diff
@@ -15,16 +15,16 @@ To regenerate code, see [codegen.cmd](codegen.cmd).
 
 The OSConfig Universal NRP binary (libOsConfigResource.so) is built with rest of OSConfig.
 
-### 3.1. Buidling the universal NRP binary on Ubuntu 14
+### 3.1. Building the universal NRP binary on Ubuntu 14
 
 The OSConfig Universal NRP binary is built on Ubuntu 14 with gcc 4.8 in order for this same binary to run as-is on all newer Linux distros.
 
 Install Ubuntu 14.04 LTS from one of the archived locations such as at [Ubuntu](https://www.releases.ubuntu.com/14.04/), or [Ubuntu MATE](https://releases.ubuntu-mate.org/archived/14.04/amd64/)). 
 
-Install the miminal set of dependencies necessary:
+Install the minimal set of dependencies necessary:
 
 ```bash
-sudo apt-get install -y gcc git cmake build-essential
+sudo apt-get install -y gcc git cmake3 build-essential

```

Comment:
>cmake3  [](http://example.com/codeflow?start=32&length=7)

Do NOT change this. This is required for the Ubuntu 14 build. Cmake 3 is only for the full OSConfig build and that is clearly stated in the main readme.

Reply:
Also fix the PR description.